chaoscenter/src/charm.py

```diff
@@ -101,12 +111,30 @@ def __init__(self, *args):
     # CONFIG METHODS #
     ##################
 
+    def _nginx_tracing_config(self) -> Optional[NginxTracingConfig]:
+        endpoint = self._workload_tracing_endpoint
```

Comment:
could you verify whether this would work too? IIRC get_endpoint already returns None if it can't find an endpoint, so perhaps we don't need to wrap it in `self._workload_tracing_endpoint` and we can call it directly.
```suggestion
        endpoint = self._workload_tracing.get_endpoint("otlp_grpc")
```